src/game_interpreter.cpp

```diff
@@ -2549,10 +2549,9 @@ bool Game_Interpreter::CommandTeleportTargets(RPG::EventCommand const& com) { //
 
 	int x = com.parameters[2];
 	int y = com.parameters[3];
-	int switch_id = (com.parameters[4] != 0)
-		? com.parameters[5]
-		: -1;
-	Game_Targets::AddTeleportTarget(map_id, x, y, switch_id);
+	bool switch_on = com.parameters[4];
```

Comment:
MSVC will complain that this is bad for performance because int is changed to bool without cast/condition

Reply:
Fixed